src/components/Editor/MonacoWrapper.tsx

```diff
@@ -28,6 +28,7 @@ export const MonacoWrapper: React.FunctionComponent<MonacoWrapperProps> = ({
     // save editor instance to the window
     window.Editor = editor;
     // parse on first run the spec
+    await EditorService.importFromURL('docs/asyncapi/asyncapi.yaml');
```

Comment:
There I have a big problem. You try to parse given mounted spec every time, so if someone will have spec saved in the localStorage, we will always override it. Due to the fact that you provided `asyncapi.yaml` in this PR, you can go to the https://deploy-preview-404--modest-rosalind-098b67.netlify.app/ preview, change something in the editor, then reload, and you will see that your changes disappeared - it's a breaking change of current solution. You can also check www.studio.asyncapi.com how now it works :)

My proposition:
- remove provided `asyncapi.yaml` in this PR
- check if given spec exist or not, by try catch - you probably need to change some logic inside `EditorService.importFromURL` function, of course I can help you with that. Let me know.
- move that line of code to the https://github.com/asyncapi/studio/blob/master/src/services/navigation.service.ts#L83 I can help with that because there are some edge cases related to the `url` and `base64` query parameters and "readOnly" mode of studio.

WDYT?